src/modules/rlm_eap/eap.c

```diff
@@ -56,7 +56,7 @@
 #define LOG_PREFIX "rlm_eap - "
 #include <freeradius-devel/modpriv.h>
 
-RCSID("$Id$")
+RCSID("$Id: 07a8041797810140f12a14d49eb1117bbc84c959 $")
```

Comment:
I don't know why this keeps happening with you, but it really breaks git for us if we merge commits with Id expanded :(